tailwind.config.ts

```
@@ -1,66 +1,69 @@
+                customGray: "#f5f5f5",
+                maroon: "#FBEAEA",
+                "maroon-600": "#A6252A",
+            },
+            borderRadius: {
+                lg: "var(--radius)",
+                md: "calc(var(--radius) - 2px)",
+                sm: "calc(var(--radius) - 4px)",
+            },
+        },
+    },
+    plugins: [require("tailwindcss-animate")],
```

ESLint

[error] A `require()` style import is forbidden.

(@typescript-eslint/no-require-imports)